src/schedulers/cron.js

```diff
@@ -1,3 +1,14 @@
+const { CronJob } = require('cron');
+
 require('./marrainageScheduler.js');
 require('./emailCreationScheduler.js');
 require('./newsletterScheduler.js');
+const { addGithubUserToOrganization } = require('./githubScheduler');
+
+module.exports.addGithubUserToOrganization = new CronJob(
```

Comment:
:thinking:  Suggestion : vu que les schedulers existants créent l'instance du `CronJob` eux-mêmes dans leur fichier (et donc gardent la spécification de la périodicité etc hors `cron.js`, qui ne fait que les importer) ne serait-il pas mieux qu'on fasse pareil pour ce _githubScheduler_ ? 

Si je comprends bien, tu décidé d'exporter la fonction `addGithubUserToOrganization` car tu la réutilises dans _scripts_, mais pourquoi pas exporter les deux ? genre :

```javascript
//  src/schedulers/githubScheduler.js
module.exports.addGithubUserToOrganization = addGithubUserToOrganization;
module.exports.addGithubUserToOrganizationCron = new CronJob()...; // Instantiation du cron
...
```

De cette façon tu peux importer _addGithubUserToOrganization_ dans scripts, et _addGithubUserToOrganizationCron_ dans `cron.js` tout en gardant l'instantiation du cron à l'intérieur du fichier comme pour les autres.

Partant pour en discuter.

Reply:
Hello @alemangui, on peut en discuter eventuellement de vive voix. Il y a plusieurs raisons pour lesquelles je suis partie là dessus : 
- je souhaitais éviter le genre de chose que l'on doit faire dans :
https://github.com/betagouv/secretariat/blob/7a29cbd32aedac14e60584362c95a280a67b7a80/tests/test-marrainage.ts#L369
`
  // Disabels global require since requiring the cron job
  // will immediatly start it.
  /* eslint-disable global-require */
  const { reloadMarrainageJob } = require('../src/schedulers/marrainageScheduler');
  // we start it manually as it may have been stopped in previous tests
  reloadMarrainageJob.start();
  clock.tick(1001);
`
Le cron est lancé à partir du moment où on require `marrainageScheduler`, et ça serait le cas pour `addGithubUserToOrganization`. Donc à partir du moment ou tu importes une fonction de ce fichier quelles qu'elles soient, le cron est créé, ce qui n'est pas vraiment clean.
Pour les tests d'ailleurs on pourrait à ce compte là aussi supprimer le `clock.tick(1001)`, `  reloadMarrainageJob.start()` un peu fastidieux et tester uniquement la fonction par exemple.
- ensuite, il  me semble aussi plus clair, de séparer la déclaration des crons des fonctions exécutées. Ces fonctions peuvent en effet être utilisé aussi dans des scripts, et on aurait d'un seul coup d'oeil les crons instanciés.